pymove/query/query.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import pandas as pd
+
+from pymove.utils import distances
+from pymove.utils.constants import DATETIME, LATITUDE, LONGITUDE, MEDP, MEDT, TRAJ_ID
+from pymove.utils.log import progress_bar
+
+
+def range_query(
+    traj,
+    move_df,
+    id=TRAJ_ID,
```

Comment:
Codacy found an issue: [Redefining built-in 'id'](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=6295868)